source/robot_model/tests/testRobotModel.cpp

```diff
@@ -1,8 +1,103 @@
 #include "robot_model/Model.hpp"
+
 #include <gtest/gtest.h>
 
+#include "robot_model/Exceptions/InvalidJointStateSizeException.hpp"
+#include "robot_model/Exceptions/FrameNotFoundException.hpp"
+
 using namespace RobotModel;
 
+class RobotModelTest : public testing::Test {
+protected:
+  void SetUp() override {
+    robot_name = "franka";
+    urdf = "/tmp/control_lib/robot_model/tests/panda_arm.urdf";
```

Comment:
Here's my suggestion for handling test fixtures.

We make the convention that test files go in module directory structure as follows: 
`robot_model/tests/fixtures/panda.urdf`

In `robot_model/CMakeLists.txt`, add the line
`target_compile_definitions(runTestRobotModel PRIVATE TEST_FIXTURES="${CMAKE_CURRENT_SOURCE_DIR}/tests/fixtures")`

Then, in this code, you can do 
`urdf = std::string(TEST_FIXTURES) + "/panda_arm.urdf";`

Because it's a private compile definition, it will be scoped to this target only. We can re-use this pattern for the other modules too.

Based on this you could remove `COPY ./source/robot_model/tests/panda_arm.urdf /tmp/control_lib/robot_model/tests/panda_arm.urdf` from the Dockerfile.

Reply:
this is a very neat solution

Reply:
very nice indeed. done